backend/src/services/worker/worker.entity.ts

```diff
@@ -7,6 +8,7 @@ class Worker {
   public passwordSalt: string;
   public tenantId: string;
   public groupIds: string[];
+  public permissions: EAMPermissionGetAllItemResponseDto[] | null;
```

Comment:
```suggestion
  public permissions: string[];
```
should be an array of string (names)

Reply:
but workers can be without group